### PR TITLE
PN-2422 - Extends microservice templates according to required dashboard metrics

### PR DESCRIPTION
**Pull request by danilosilvase**
- add dashboard fro microsevice pn-logsaver-pe

### CODE REVIEW

**Review by fdammassa — CHANGES_REQUESTED** [review]
@danilosilvase please enable the alarms on batch job failure

**Comment by danilosilvase** [comment]
> @danilosilvase please enable the alarms on batch job failure



> @danilosilvase please enable the alarms on batch job failure

thanks for review it. modification done.

File: scripts/aws/cfn/microservice.yml [hunk 1]
```diff
@@ -440,25 +444,23 @@ Resources:
...
+         - aws.batch
+       detail-type:
+         - Batch Job State Change
+       detail:
+         status:
+           - FAILED
+     State: ENABLED
+     Targets:
+       - Arn: !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:${AlarmSNSTopicName}"
+         Id: !Sub "${ProjectName}-logsaver-be"
```
Comment (fdammassa) [inline scripts/aws/cfn/microservice.yml:463]: @danilosilvase what does "Id" represent?

File: scripts/aws/cfn/microservice.yml [hunk 2]
```diff
@@ -135,6 +135,10 @@ Parameters:
     Default: 1
     Description: The maximum number of batch job attempts.
 
+  AlarmSNSTopicName:
```
Comment (fdammassa) [inline scripts/aws/cfn/microservice.yml:138]: @danilosilvase please use `AlarmSNSTopicArn`

**Review by fdammassa — CHANGES_REQUESTED** [review]

File: scripts/aws/cfn/microservice.yml (hunk 1 above)
Comment (danilosilvase) [reply scripts/aws/cfn/microservice.yml:463]: This ID is used to represent the target name within the specified rule. 

something like this:
Screenshot 2022-10-26 at 11 47 54

**Review by fdammassa — APPROVED** [review]

**Review by marco-vit-pagopa — APPROVED** [review]

**Pull request merged**